libs/ui-watt/src/lib/components/table/watt-table.component.html

```diff
@@ -80,10 +80,10 @@
     mat-row
     *matRowDef="let row; columns: _getColumns()"
     [attr.aria-selected]="row === activeRow"
-    (click)="rowClick.emit(row)"
+    (click)="disabled ? null : rowClick.emit(row)"
```

Comment:
suggestion: move this to class method, something like `_onRowClick`, seems a bit wrong to return null here IMO.